src/Native/Windows/build-native.cmd

```diff
@@ -10,36 +10,53 @@ set __IntermediatesDir=""
 set __BuildArch=AnyCPU
 set __VCBuildArch=x86_amd64
 set CMAKE_BUILD_TYPE=Release
+set __EnvironmentSet=false
+set __VSVersion=vs2013
```

Comment:
Thanks @jkotas, I'll change it to vs2015.

At some point we should look into consolidating the cmake bits between coreclr and corefx a la https://github.com/dotnet/coreclr/issues/1344. The CoreFX and CoreCLR windows native builds used to use the exact same code, so we could certainly get back to that point again in the future. That's a discussion we can continue in that issue, though.